indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -36,9 +36,17 @@ export default function FormItem({
       <div styleName="content">
         {InputComponent ? (
           customFormItem ? (
-            <InputComponent isRequired={isRequired || meta.alwaysRequired} {...inputProps} />
+            <InputComponent
+              isRequired={isRequired || meta.alwaysRequired}
+              disabled={!isEnabled}
```

Comment:
No need to pass a value for it here - `!isEnabled` is for fully disabling a field, but eventually (when there's the final-form integration) the disabled flag for the field that's being displayed will be provided from outside (by final-form), and not taken from this. That said I think for the preview in management there's nothing wrong with disabling the field... when testing it, it actually looked kind of nice so feel free to leave it :)